.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: "#{build}"
+pull_requests:
+  do_not_increment_build_number: true
+configuration:
+  - Release
+clone_depth: 5
+build_script:
+  - cmd: call .appveyor\appveyor_helper.bat
```

Comment:
So what happens when appveyor_helper.bat exits with an error? would it still continue to build PBS?